lib/vrl/cli/src/repl.rs

```diff
@@ -10,21 +11,29 @@ use rustyline::{Context, Editor, Helper};
 use std::borrow::Cow::{self, Borrowed, Owned};
 use vrl::{diagnostic::Formatter, state, Runtime, Target, Value};
 
+// Create a list of all possible error values for potential docs lookup
+lazy_static! {
+    static ref ERRORS: Vec<String> = (100..=110).map(|i| i.to_string()).collect();
+}
+
 const HELP_TEXT: &str = r#"
 VRL REPL commands:
-  help functions    Display a list of currently available VRL functions (aliases: ["help funcs", "help fs"])
-  help docs         Navigate to the VRL docs on the Vector website
-  help docs <func>  Navigate to the VRL docs for the specified function
-  next              Load the next object or create a new one
-  prev              Load the previous object
-  exit              Terminate the program
+  help functions     Display a list of currently available VRL functions (aliases: ["help funcs", "help fs"])
+  help docs          Navigate to the VRL docs on the Vector website
+  help docs <func>   Navigate to the VRL docs for the specified function
+  help error <code>  Navigate to the docs for a specific error code
+  next               Load the next object or create a new one
+  prev               Load the previous object
+  exit               Terminate the program
 "#;
 
 const DOCS_URL: &str = "https://vector.dev/docs/reference/vrl";
+const ERRORS_URL_ROOT: &str = "https://errors.vrl.dev";
```

Comment:
There is no webpage, so I'm guessing there are no docs for errors at the moment ?.

Reply:
Not until the 0.12 docs are deployed